### DISCUSSION TITLE
lingua-js Release

### DISCUSSION THREAD

**Discussion by thirtha-prasad** [pemistahl/lingua-js#1] [General]
Hi Any update on when we can expect the javascript version of lingua to be released

**Comment by pemistahl** [comment]
Not yet, I'm sorry. I do all of this in my spare time which is currently very limited due to my family and my daily job. I will continue to work on the project but I can't tell you when.

**Reply by thirtha-prasad** [reply]
Thank you for the reply, we are currently using lingua-py in our automation which is done in cypress. As cypress is a JavaScript based automation it would be great if we can have this in JS soon.

**Reply by thirtha-prasad** [reply]
Or with whatever is done till now can we use it for detecting language.
